source/tutorials/nix-language.md

```diff
@@ -1489,8 +1494,16 @@ The file is copied into the Nix store directory `/nix/store` as a side effect of
 
 It is an error if the file system path does not exist.
 
+:::{note}
+Note that the Nix language assumes that all input files will remain _unchanged_ during the course of the Nix expression evaluation.
+If you for example antiquote a file path during a `nix repl` session, and then later in the same session, after having changed the file contents, evaluate the antiquotation with the file path again, then Nix will still return the first store path.
+It will _not_ reread the file contents to produce a different Nix store path.
+:::
+
```

Comment:
```suggestion
```

Sorry for the back and forth, but I thought about it again and came to the conclusion that this is too much of an implementation detail. This belongs to the Nix language reference. Would be great if you added your note [somewhere around here](https://github.com/NixOS/nix/blob/a324e9a5c84a144b824303064220463977c63c73/doc/manual/src/language/values.md?plain=1#L146-L151).

Reply:
https://github.com/NixOS/nix/pull/7191 👍🏻